### DISCUSSION TITLE
Trouble Installing / Lag Issues with Tombs of Amascut

### DISCUSSION THREAD

**Discussion by bokaeh** [runelite/runelite#15566] [Q&A]
Jagex recently stated that possible fix for a lag issue within Raids 3 could be updating the RuneLite Launcher. However after installing the latest version of RL for my computer (Macbook running macOS Monterrey12.5), there was an error saying this version is not supported by my Mac. 
I luckily saved the older launcher and can play but I am still running into the lag issue within Raids 3. More specifically I encounter the lag as I enter the Puzzle Path for the Kalphite Room for the second or third raid without reseting the client.

Am I missing something?

**Comment by Adam-** [comment]
MacOS Monterey is supported, the latest launcher supports 10.14+ which is Mojave and newer

**Reply by bokaeh** [reply]
Huh, then I am not sure why the newest launcher isn't compatible with my Mac.
